packages/devtools_app/lib/src/utils.dart

```diff
@@ -420,3 +420,21 @@ bool isDebugBuild() {
   })());
   return debugBuild;
 }
+
+/// Divides [numerator] by [denominator], not returning infinite, NaN, or null
+/// quotients.
+///
+/// Returns [ifNotFinite] as a return value when the result of dividing
+/// [numerator] by [denominator] would be a non-finite value: either
+/// NaN, null, or infinite.
+///
+/// [ifNotFinite] defaults to 0.0.
+double safeDivide(num numerator, num denominator, [double ifNotFinite = 0.0]) {
```

Comment:
a named parameter for the value on divide failure might be more intuitive.
ideas for a parameter name:
`failureValue`, `IfNotFinite`, `ifInvalid`

Reply:
Done.